### PR TITLE
feat(*): implement draft connect

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

File: cmd/draft/connect.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	connection, err := deployedApp.Connect(clientset, clientConfig)
+	if err != nil {
+		return err
+	}
+
+	detail := fmt.Sprintf("localhost:%#v", connection.Tunnel.Local)
+	fmt.Fprintln(cn.out, "SUCCESS...Connect to your app on "+detail)
+
+	fmt.Fprintln(cn.out, "Starting log streaming...")
+	readCloser, err := connection.RequestLogStream(deployedApp)
```
Comment (bacongobbler) [inline cmd/draft/connect.go]: we need to check if err != nil after this block

File: cmd/draft/create.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ import (
 )
 
 const (
+	draftToml  = "draft.toml"
```
Comment (bacongobbler) [inline cmd/draft/create.go]: nice 👍

File: pkg/draft/context.go [hunk 3]
```diff
@@ -48,8 +48,8 @@ func newAppContext(s *Server, req *rpc.UpRequest, out io.Writer) (*AppContext, e
 
 	// inject certain values into the chart such as the registry location,
 	// the application name, and the application version.
-	tplstr := "image.name=%s,image.org=%s,image.registry=%s,image.tag=%s,basedomain=%s,ondraft=true"
-	inject := fmt.Sprintf(tplstr, req.AppName, s.cfg.Registry.Org, s.cfg.Registry.URL, imgtag, s.cfg.Basedomain)
+	tplstr := "image.name=%s,image.org=%s,image.registry=%s,image.tag=%s,basedomain=%s,ondraft=true,draft=%s"
```
Comment (bacongobbler) [inline pkg/draft/context.go]: Would it be better to reference `local.DraftLabelKey` here (and make `local.draftLabelKey` public as `local.DraftLabelKey`), or do we not want third party libraries to use that package?

File: pkg/draft/local/local_test.go [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+		t.Fatal(err)
+	}
+
+	if !reflect.DeepEqual(expectedApp, app) {
+		t.Errorf("Expected %#v, got %#v", expectedApp, app)
+	}
+}
+
+func TestConnect(t *testing.T) {
+	t.Skip()
```
Comment (bacongobbler) [inline pkg/draft/local/local_test.go]: just skipping this test for now to get this out the door?

**Comment by bacongobbler** [comment]
Oh, let's also make sure to capture a new ticket for the `draft connect` only showing logs and tunnelling from the first container in the pod.

File: cmd/draft/connect.go (hunk 1 above)
Comment (michelleN) [reply cmd/draft/connect.go]: good catch!

**Comment by bacongobbler** [comment]
Works as advertised. Fantastic work, michelle!!

```
><> draft up
Draft Up Started: 'ordered-dog'
ordered-dog: Building Docker Image: ⚓  (48.0741s)
ordered-dog: Pushing Docker Image: ⚓  (50.0625s)
ordered-dog: Releasing Application: ⚓  (0.3760s)
ordered-dog: Build ID: 01BRZ7VSGKQ5R9VGF5MPJQG3BQ
><> draft connect
Connecting to your app...Error: could not find a ready pod
><> draft connect
Connecting to your app...Error: could not find a ready pod
><> k get po
NAME                                 READY     STATUS              RESTARTS   AGE
ordered-dog-python-147009253-mgrf1   0/1       ContainerCreating   0          17s
ordered-dog-python-147009253-vd29k   0/1       ContainerCreating   0          17s
><> sleep 30
><> k get po
NAME                                 READY     STATUS              RESTARTS   AGE
ordered-dog-python-147009253-mgrf1   1/1       Running             0          55s
ordered-dog-python-147009253-vd29k   0/1       ContainerCreating   0          55s
><> draft connect
Connecting to your app...SUCCESS...Connect to your app on localhost:55225
Starting log streaming...
 * Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
172.17.0.1 - - [01/Sep/2017 17:32:48] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [01/Sep/2017 17:32:48] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [01/Sep/2017 17:32:58] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [01/Sep/2017 17:32:58] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [01/Sep/2017 17:33:08] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [01/Sep/2017 17:33:08] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [01/Sep/2017 17:33:15] "GET / HTTP/1.1" 200 -
172.17.0.1 - - [01/Sep/2017 17:33:18] "GET / HTTP/1.1" 200 -
```

**Pull request merged**